drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableConstants.properties

```diff
@@ -36,6 +36,7 @@ ValueList=Value list
 ValueListsExplanation=Value lists are an optional comma separated list of values to show as a drop down.
 optionalValueList=(optional) value list:
 DefaultValue=Default value
+DefaultValueExplanation=Change of ''Default value'' will have effect just on new table rows.
```

Comment:
Proposal: to change this title on  "Change of ''Default" value affects only the new table rows."

Reply:
Thank you, applied